app/assets/stylesheets/home/pages/_index.sass

```diff
@@ -21,3 +21,6 @@
   +for-desktop-up
     padding-left: 5.5rem
     padding-right: 5.5rem
+
+.home__flow > * + *
```

Comment:
what does this do? I haven't seen it before

Reply:
https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/

Reply:
Mind blown!